### PR TITLE
[ErrorHandler] Skip failing tests when "xdebug.file_link_format" option is defined

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        |
| License       | MIT

This test annoys me since ... many years!
So let's skip it.

### CODE REVIEW

File: src/Symfony/Component/ErrorHandler/Tests/ErrorRenderer/FileLinkFormatterTest.php [hunk 1]
```diff
@@ -27,6 +27,11 @@ public function testWhenNoFileLinkFormatAndNoRequest()
 
     public function testAfterUnserialize()
     {
+        if (get_cfg_var('xdebug.file_link_format')) {
```
Comment (stof) [inline src/Symfony/Component/ErrorHandler/Tests/ErrorRenderer/FileLinkFormatterTest.php:30]: shouldn't this use `ini_get` instead ?

File: src/Symfony/Component/ErrorHandler/Tests/ErrorRenderer/FileLinkFormatterTest.php [hunk 2]
```diff
@@ -27,6 +27,11 @@ public function testWhenNoFileLinkFormatAndNoRequest()
...
+            // There is not way to override "xdebug.file_link_format" option in a test.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/ErrorHandler/Tests/ErrorRenderer/FileLinkFormatterTest.php:31]:
```suggestion
            // There is no way to override "xdebug.file_link_format" option in a test.
```

File: src/Symfony/Component/ErrorHandler/Tests/ErrorRenderer/FileLinkFormatterTest.php (hunk 1 above)
Comment (lyrixx) [reply src/Symfony/Component/ErrorHandler/Tests/ErrorRenderer/FileLinkFormatterTest.php:30]: Nope. I don't use xdebug. So the param is not available via ini_get. See the implementation of the file link formatter

**Comment by nicolas-grekas** [comment]
Thank you @lyrixx.

**Pull request merged**